### PR TITLE
[PR] Structure

### PR DESCRIPTION
**Pull request by philcable**
Working on issues 1-3, but these first five commits are for #1.

@jeremyfelt Please #review when you get a chance. Let me know if the naming conventions and plugin structure are agreeable so far!

### CODE REVIEW

**Comment by philcable** [comment]
Fixes #1.

@jeremyfelt Since it might be a day or two before I get back to it, this is as good a time as any for a #reviewmerge.

File: includes/class-people-display-admin-list-table.php [hunk 1]
```diff
@@ -0,0 +1,240 @@
+<?php
+/**
+ * `WP_List_Table` is technically private, so using it may be frowned upon.
```
Comment (jeremyfelt) [inline includes/class-people-display-admin-list-table.php:3]: Sometimes you have to do what you have to do. :tada:

File: includes/class-people-display-admin-list-table.php [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+			'name' => 'Name',
+			// The following two are just for testing purposes.
+			'email' => 'Email',
+			'classification' => 'Classification',
+		);
+
+		$user = get_current_user_id();
+		$screen = get_current_screen();
+		$option = $screen->get_option( 'per_page', 'option' );
+		$per_page = get_user_meta( $user, $option, true );
```
Comment (jeremyfelt) [inline includes/class-people-display-admin-list-table.php:122]: I could be missing something, but where are these variables being used after assignment?

**Review by jeremyfelt — CHANGES_REQUESTED** [review]
I made a couple comments inline, but nothing that needs to be changed before merge. I have another comment that I'll open as an issue instead for a future revision. And I opened #8 with some changes to this branch before we merge.

**Review by jeremyfelt — APPROVED** [review]

**Comment by jeremyfelt** [comment]
Ok, merging! Great start, @philcable!

**Pull request merged**